components/menu/stories/menu.stories.js

```diff
@@ -375,9 +406,147 @@ Collapsible.args = {
+			description: "Short description of menu item",
+		},
+		{
+			idx: 4,
+			label: "Menu item with value",
+			value: "Value",
+		},
+		{
+			idx: 5,
+			label: "Menu item with icon and description",
+			description: "Short description of menu item",
+			iconName: "Cloud",
+		},
+	]
+};
+Sizing.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * When a menu item contains a submenu, a drill-in chevron will appear at the end of the menu item to show that a
+ * submenu is available.
+ */
+export const DrillInChevron = Template.bind({});
+DrillInChevron.storyName = "Submenu drill-in chevron";
+DrillInChevron.tags = ["!dev"];
+DrillInChevron.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+DrillInChevron.args = {
+	items: [
+		{
+			label: "Menu item (with submenu)",
+			isDrillIn: true,
+		},
+		{
+			label: "Menu item",
+		},
+		{
+			label: "Menu item",
+		}
+	],
+};
+
+export const PopoverSubmenu = SubmenuInPopover.bind({});
+PopoverSubmenu.storyName = "Submenu in popover";
+PopoverSubmenu.tags = ["!dev"];
+PopoverSubmenu.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * A menu section has the options of single selection, multiple selection, or having no selection. By default, menu
+ * items have no selection, and perform an action on press.
+ *
+ * For single selection menu sections, menu items show a single checkmark to indicate the selected item. Multiple
+ * selection menu sections display checkboxes or switches beside each menu item.
+ */
+export const MenuItemSelection = SelectionGroup.bind({});
+MenuItemSelection.storyName = "Selection of menu items";
+MenuItemSelection.tags = ["!dev"];
+MenuItemSelection.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * The last item in each of these menus is disabled. A menu item in a disabled state shows that an option exists, but
+ * is not available in that circumstance. This state can be used to maintain layout continuity and to communicate that
+ * an action may become available later.
+ */
+export const DisabledItems = DisabledItemGroup.bind({});
+DisabledItems.storyName = "Disabled items";
+DisabledItems.tags = ["!dev"];
+DisabledItems.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * When a menu item’s label or description exceed the available horizontal space, the default behavior is to wrap the
+ * text to a new line.
+ *
+ * Menu item labels and headings can be truncated with an ellipsis by using the `spectrum-Menu-itemLabel--truncate`
+ * class. This class should be applied to the item label or section heading that requires truncation within a menu with
+ * a set `inline-size` or `max-inline-size`. When text is truncated, it should be revealed with a tooltip on hover
+ * (not displayed here).
+ *
+ * Truncation can be used in conjunction with icons, values, and drill-ins in a menu item. Note that descriptions
+ * within a menu item always wrap by default and do not have a truncation option.
+ */
```

Comment:
```suggestion
/**
 * When a menu item’s label or description exceed the available horizontal space, the default behavior is to wrap the
 * text to a new line.
 *
 * Menu item labels and headings can be truncated with an ellipsis by using the `spectrum-Menu-itemLabel--truncate`
 * class. Truncation can only occur if the menu has
 * a set `inline-size` or `max-inline-size`, or it is constrained by the width of its parent element(s). For demonstration purposes, a `max-inline-size` is set on the following menu examples. When text is truncated, it should be revealed with a tooltip on hover
 * (not displayed here).
 *
 * Truncation can be used in conjunction with icons, values, and drill-ins in a menu item. Note that descriptions
 * within a menu item always wrap by default and do not have a truncation option.
 */
```
It doesn't necessarily need an `inline-size` or `max-inline-size`. It could also be constrained by the width of its parent. We could also drop that sentence entirely instead of trying to explain all situations where text overflow would occur. I think it's worth noting when we are setting inline styles to show certain behavior in the docs, so I also added a sentence for that.